System_monitor build failure on Nvidia Xavier platform

### Checklist

- [X] I've read the [contribution guidelines](https://github.com/autowarefoundation/autoware/blob/main/CONTRIBUTING.md).
- [X] I've searched other issues and no duplicate issues were found.
- [X] I'm convinced that this is not my fault but a bug.

### Description

system_monitor reported error building on Xavier.

```
--- stderr: system_monitor                                                                                                                                                   
/home/nv/ws/autoware/src/universe/autoware.universe/system/system_monitor/src/gpu_monitor/tegra_gpu_monitor.cpp: In member function ‘virtual void GPUMonitor::checkThrottling(diagnostic_updater::DiagnosticStatusWrapper&)’:
/home/nv/ws/autoware/src/universe/autoware.universe/system/system_monitor/src/gpu_monitor/tegra_gpu_monitor.cpp:130:80: error: unused parameter ‘stat’ [-Werror=unused-parameter]
  130 | void GPUMonitor::checkThrottling(diagnostic_updater::DiagnosticStatusWrapper & stat)
      |                                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~
cc1plus: all warnings being treated as errors
make[2]: *** [CMakeFiles/gpu_monitor_lib.dir/build.make:76: CMakeFiles/gpu_monitor_lib.dir/src/gpu_monitor/tegra_gpu_monitor.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:602: CMakeFiles/gpu_monitor_lib.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
---
Failed   <<< system_monitor [9min 1s, exited with code 2]
```

### Expected behavior

Build success

### Actual behavior

Build error

### Steps to reproduce

Build

### Versions

- OS: Ubuntu 20.04
- ROS2: Galactic
- Autoware: main with Docker

### Possible causes

Unused variable

### Additional context

_No response_

--- a/system/system_monitor/src/gpu_monitor/tegra_gpu_monitor.cpp
+++ b/system/system_monitor/src/gpu_monitor/tegra_gpu_monitor.cpp
@@ -127,7 +127,8 @@ void GPUMonitor::checkUsage(diagnostic_updater::DiagnosticStatusWrapper & stat)
   }
 }
 
-void GPUMonitor::checkThrottling([[maybe_unused]]diagnostic_updater::DiagnosticStatusWrapper & stat)
+void GPUMonitor::checkThrottling(
+  [[maybe_unused]] diagnostic_updater::DiagnosticStatusWrapper & stat)
 {
   // Please emove the [[maybe_unused]] tag after implementation, it's a temp build fix
   // TODO(Fumihito Ito): implement me

--- a/system/system_monitor/src/gpu_monitor/tegra_gpu_monitor.cpp
+++ b/system/system_monitor/src/gpu_monitor/tegra_gpu_monitor.cpp
@@ -130,7 +130,7 @@ void GPUMonitor::checkUsage(diagnostic_updater::DiagnosticStatusWrapper & stat)
 void GPUMonitor::checkThrottling(
   [[maybe_unused]] diagnostic_updater::DiagnosticStatusWrapper & stat)
 {
-  // Please emove the [[maybe_unused]] tag after implementation, it's a temp build fix
+  // Please remove the [[maybe_unused]] tag after implementation, it's a temp build fix
   // TODO(Fumihito Ito): implement me
 }
 

--- a/system/system_monitor/src/gpu_monitor/tegra_gpu_monitor.cpp
+++ b/system/system_monitor/src/gpu_monitor/tegra_gpu_monitor.cpp
@@ -127,8 +127,9 @@ void GPUMonitor::checkUsage(diagnostic_updater::DiagnosticStatusWrapper & stat)
   }
 }
 
-void GPUMonitor::checkThrottling(diagnostic_updater::DiagnosticStatusWrapper & stat)
+void GPUMonitor::checkThrottling([[maybe_unused]]diagnostic_updater::DiagnosticStatusWrapper & stat)
 {
+  // Please emove the [[maybe_unused]] tag after implementation, it's a temp build fix
   // TODO(Fumihito Ito): implement me
 }
 

